indungu/yummy-recipes
# [FIX] Added some Custom for a more aesthetic UI for better UX

### What this PR does
Adds some functionality to the UI on the `dashboard`

### Where to begin your review

1. The changes are yet to be reflected on the current live static UI you will find [here](https://indungu.githu.io/yummy-recipes) 
2. Navigate through the site noting the UI changes made to the `welcome`,`signup` and `login` pages.

### How/What to test

Follow the following steps to set up your testing:
```
  1. Clone the repo's `dev` branch locally: git clone -b dev https://github.com/indungu/yummy-recipes
  2. Change directory: cd yummy-recipes 
  3. Run the file in the designs/UI/index.html 
  4. Check if the above mentioned buttons are present.
```
### Notes
When this PR is merged it should solve (from a Static UI perspective) to do item `2` in the To-Do 
section of the `master` *README.md*
The changes made from [this](https://github.com/indungu/yummy-recipes/pull/2) PR are yet to be migrated to the `gh-pages` branch which is the branch from which the site provided in the *Where to begin* section is served from, do not be alarmed.

### Questions
In your own personal opinion, how much more aesthetic appeal did the changes in this PR add? 
*(on a scale of 1 to 5 where 5 is _A Whole LOT_)*
What  further changes would you propose?
